spec.bs

```diff
@@ -1717,3 +1764,145 @@ specification is printed below:
+  [=Document/permissions policy=].
+
+  Modify step 1 of the algorithm to read:
+  1. Let <var ignore>policy</var> be the result of running
+    [=Create a Permissions Policy for a navigable=] given |container|, |origin|, and
+    |required features|.
+</div>
+
+<div algorithm=shared-document-creation-changes>
+  Modify the [=create and initialize a Document object=] algorithm. Rewrite step 3 to read:
+
+  3. Let <var ignore>permissionsPolicy</var> be the result of [=Create a Permissions Policy for a
+    navigable from response|creating a permissions policy from a response=] given
+    |navigationParams|'s [=navigable=]'s [=navigable container|container=], |navigationParams|'s
+    [=navigation params/origin=], |navigationParams|'s [=navigation params/response=], and
+    |navigationParams|'s [=navigable=]'s [=fenced frame config instance=]'s [=fenced frame config
+    instance/required permissions to load=].
+</div>
+
+<div algorithm=new-browsing-context-changes>
+  Modify the [=create a new browsing context and document=] algorithm. Rewrite step 7 to read:
+
+  7. Let <var ignore>permissionsPolicy</var> be the result of [=Create a Permissions Policy for a
+    navigable|creating a permissions policy=] given <var ignore>embedder</var>,
+    <var ignore>origin</var>, and null.
+
+  Note: This change is made in addition to the changes to [=create a new browsing context and
+  document=] outlined in [[#creating-browsing-contexts-patch]].
+</div>
+
+<div algorithm=attempt-populate-history-patches>
+  Modify [[HTML]]'s [=attempt to populate the history entry's document=] algorithm. Add a step
+  before the step inside the [=queue a task|queued task=] starting with "If
+  |failure| is true, then:" that reads:
+
+  8. Otherwise, if the result of [=Should navigation response to navigation request be blocked by
+    Permissions Policy?=] given <var ignore>navigationParams</var>'s [=response=], and <var
+    ignore>navigable</var> is "`Blocked`", then set |failure| to true.
+</div>
+
+<div algorithm=permissions-policy-block-request>
+  Create a new algorithm called <dfn>Should navigation response to navigation request be blocked by
+  Permissions Policy?</dfn> in [[!HTML]].
+
+  Given a [=response=] (|response|) and a [=navigable=] (|navigable|), this algorithm returns
+  `Blocked` or `Allowed`:
+
+  1. If |navigable| is not a [=fenced navigable container/fenced navigable=], then return `Allowed`.
+
+  2. Let |required permissions to load| be the |navigable|'s [=navigable/fenced frame config
+     instance=]'s [=fenced frame config instance/required permissions to load=].
+
+  3. Let |inherited policy| be |navigable|'s [=navigable/active document=]'s [=Document/permissions
+     policy=]'s [=inherited policy=].
+
+  4. Let |origin| be |response|'s [=response/url=]'s [=url/origin=].
+
+  5. [=list/For each=] |permission| of |required permissions to load|:
+
+    1. If |permission| does not exist in |inherited policy|, and |permission|'s
```

Comment:
My understanding is that `inherited policy` will be completely full of all supported features, so it should always exist, per https://w3c.github.io/webappsec-permissions-policy/#inherited-policy-for-a-feature.

Do you mean to check if `|inherited policy|[|permission|] is "Disabled"` instead?